sdk/url.js

```diff
@@ -59,6 +59,14 @@ const getStatsUrl = function getStatsUrl(accountId, apiKey) {
   return urljoin(`https://${getDeployEndpoint()}`, 'v3', 'metrics', accountId);
 };
 
+function getTriggersListUrl() {
+  return urljoin(`https://${getDeployEndpoint()}`, 'v3', 'triggers');
+}
+
+function getSpecificTriggerUrl(triggerName) {
+  return urljoin(`https://${getDeployEndpoint()}`, 'v3', 'trigger', triggerName);
```

Comment:
It might be slightly drier to have a base endpoint function for just the `https://${getDeployEndpoint()}`, 'v3' part

Reply:
don't want to refactor the whole cli right now :\

Reply:
I don't know how that requires a whole CLI refactor, but it's minimal anyway

Reply:
sorry, I've miss read your comment